lib/galaxy/util/sanitize_html.py

```diff
@@ -112,17 +114,17 @@ def unknown_starttag(self, tag, attrs):
                 value = value.replace('>', '&gt;').replace('<', '&lt;').replace('"', '&quot;')
                 value = self.bare_ampersand.sub("&amp;", value)
                 # thanks to Kevin Marks for this breathtaking hack to deal with (valid) high-bit attribute values in UTF-8 feeds
-                if isinstance(value, unicode):
+                if isinstance(value, text_type):
                     try:
-                        value = unicode(value, self.encoding)
-                    except:
-                        value = unicode(value, 'iso-8859-1')
-                uattrs.append((unicode(key, self.encoding), value))
+                        value = unicodify(value, self.encoding)
+                    except Exception:
+                        value = unicodify(value, 'iso-8859-1')
```

Comment:
If you guys want to continue with patching feedparser and creating a Galaxy dependency you sure can. Seems like a lot of effort to me though and this PR may well stale in the process. As long as this doesn't break the Python 2 version - I'd just do your best with the code right here and create issues for:

 - [ ] Creating an external dependency on feedparser or a Galactic fork.
 - [ ] Add a TODO item for fixing and testing this piece of code in Python 3 as part of #1715.